src/main/java/bleuauction/bleuauction_be/server/notice/controller/NoticeController.java

```diff
@@ -150,57 +94,27 @@ public ResponseEntity<?> fileNoticeDelete(@RequestHeader("Authorization") String
     }
   }
 
-
-
-  //디테일(수정)
-  @GetMapping("/api/notice/detail/{noticeNo}")
-  public ResponseEntity<Notice> detailNotice(@PathVariable("noticeNo") Long noticeNo) {
-    Notice notice = noticeService.findOne(noticeNo);
-
-    if (notice != null) {
-      notice.setNoticeAttaches(notice.getNoticeAttaches());
-      return ResponseEntity.ok(notice);
-    } else {
-      return ResponseEntity.notFound().build();
-    }
+  //디테일
+  @GetMapping("/detail/{noticeNo}")
+  public Notice detailNotice(@PathVariable("noticeNo") Long noticeNo) {
```

Comment:
여기 Mapping은 전 이렇게 했을 것 같아요

```suggestion
  @GetMapping("/{noticeNo}")
  public Notice detailNotice(@PathVariable("noticeNo") Long noticeNo) {
```